content/sensu-go/6.1/observability-pipeline/observe-schedule/subscriptions.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Subscriptions reference"
+linkTitle: "Subscriptions Reference"
+reference_title: "Subscriptions"
+type: "reference"
+description: "Sensu subscriptions are equivalent to topics in a traditional publish/subscribe system. With Sensu subscriptions, you can configure check requests in a one-to-many model for an entire group or subgroup of systems rather than a traditional one-to-one mapping of configured hosts or observability checks. Read this reference doc to learn how to configure subscriptions in Sensu."
+weight: 35
+version: "6.1"
+product: "Sensu Go"
+platformContent: false
+menu:
+  sensu-go-6.1:
+    parent: observe-schedule
+---
+
+Sensu uses the [publish/subscribe pattern of communication][1].
+At the core of this model are Sensu agent and check subscriptions, which are equivalent to "topics" in a traditional publish/subscribe system.
+Sensu entities become subscribers to these "topics" via their individual `subscriptions` attributes.
+Sensu checks also have a `subscriptions` attribute, where you specify which subscribers will execute the checks.
+The Sensu backend schedules checks and publishes check execution requests to entities.
+
+Subscriptions are powerful service-based monitoring primitives that allow you to effectively monitor for specific behaviors or characteristics that correspond to the function provided by a particular system.
```

Comment:
```suggestion
Subscriptions are powerful service-based monitoring primitives that allow you to effectively monitor for specific behaviors or characteristics that correspond to the function provided by a particular system.
```

This statement is true, but really doing much for me. Either here or earlier in the document, we need to better explain how the loose coupling used by publish/subscribe model -- and therefore subscriptions -- are beneficial. 

A few thoughts:

Publish/subscribe is important for ephemeral or elastic infrastructure where the names of things and number of things change over time.

Using a publish/subscribe model, humans writing checks don't need to know the specific names of the entities that will run the check.

Conversely, entities do not need to know the specific check names they should execute. 

The backend correlates overlapping subscriptions between checks and entities to determine which entities should receive execution requests for a given check.

Reply:
Maybe part of what I'm missing here is enough explanation of pub/sub that an unfamiliar reader will understand the connection to the next sentence?